Tasks/AzureRmWebAppDeployment/operations/KuduServiceUtility.ts

```diff
@@ -34,38 +34,40 @@ export class KuduServiceUtility {
         }
     }
 
-    public async runPostDeploymentScript(taskParams: TaskParameters): Promise<void> {
+    public async runPostDeploymentScript(taskParams: TaskParameters, directoryPath?: string): Promise<void> {
         try {
+            directoryPath = (!!directoryPath) ? directoryPath : physicalRootPath.substring(1);
```

Comment:
what does physicalRootPath.substring(1) do?

Reply:
In api/command API, we pass the command and the path (where we need to run ).
Path shouldn't contain trailing slash, if it contains then it considered as absolute path in Linux.
physicalRootPath contains slash prefix and we are removing it.